microsoft-edge/devtools-guide-chromium/reasons-edge-devtools.md

```diff
@@ -0,0 +1,215 @@
+
+Console with highlighted search icon that would open a web search with the error message in a new browser tab:
+
+![Console with highlighted search icon that would open a web search with the error message in a new browser tab.](reasons-edge-devtools-images/console-error-message-search-web-link.png)
+
+See [Search the web for a Console error message string](console/index.md#search-the-web-for-a-console-error-message-string) in _Console overview_.
+
+
+### 3D View
+
+The **3D View** tool displays either the DOM, the composited layers of a webpage, or the parts of a webpage page that have a z-index assigned to them, as a 3-dimensional representation in an interactive user interface.  This dynamic view allows you to gauge the complexity of your webpage and find out about memory problems.
+
+![The 3D View tool.](reasons-edge-devtools-images/3d-tool.png)
+
+See [Navigate z-index, DOM, and layers using the 3D View tool](3d-view/index.md).
+
+
+### Integration with webhint
+
+The **Issues** tool automatically shows you all the things that need fixing in the current webpage.  Under the hood, the **Issues** tool uses [webhint](https://webhint.io), which generates reports and provides reporting on various issues.  You not only learn what's wrong with your webpage, but also why it's a problem, where in your webpage the problem exists, and how to fix the issue.
+
+Current problems of a webpage displayed in the Issues panel:
+
+![Current problems of a webpage displayed in the Issues panel.](reasons-edge-devtools-images/webpage-problems-displayed-in-issues-panel.png)
+
+See [Find and fix problems using the Issues tool](issues/index.md).
+
+
+### Integration of Microsoft Edge DevTools with Visual Studio Code
+
+When you're debugging a problem with your webpage, it's cumbersome to change from a code editor, to the browser, then using DevTools in the browser to analyze the problem, and then going back to try a possible fix of the problem in your source code.  This is why we created a Visual Studio Code extension that embeds Microsoft Edge DevTools into your code editor workflow. 
+
+For example, without integrated tools, you temporarily fix CSS style rules in your transient, resulting webpage files that are sent from your development server to your browser, and then you have to remember the changes, and then re-apply those changes to your actual source code in your source files, over in your code editor.
+
+Now, by using the Microsoft Edge DevTools extension for Visual Studio Code, you can simply use Microsoft Edge DevTools to change your source code directly within an instance of DevTools that's running in the IDE environment of Visual Studio Code.<!-- condense end -->
+
+Microsoft Edge DevTools integrated into Visual Studio Code:
+
+![Microsoft Edge DevTools integrated into Visual Studio Code.](reasons-edge-devtools-images/devtools-integrated-into-visual-studio-code.png)
```

Comment:
It might be better for this screenshot to also show code next to devtools. Would you be able to redo the screenshot with the file `base.css` opened in VSCode. The ideal layout we want to show here is 3 columns:

1. base.css code editor
2. devtools elements panel
3. devtools screencast browser

If it causes the columns to become too narrow, then I'd suggest collapsing the **Explorer** sidebar to save some space.